.github/workflows/build.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        go: [ '1.19.1', '1.18.6' ]
+        go: [ '1.23.0' ]
```

Comment:
Here is a better update to this file:

```yaml
on: [ push, pull_request ]
name: Build
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v4

      - name: Install Go
        uses: actions/setup-go@v5
        with:
          go-version-file: 'go.mod'
          check-latest: true
          cache: true

      - name: Set Hosts
        run: |
          echo "127.0.0.1	example.com example2.com" | sudo tee -a /etc/hosts

      - name: Test
        run: |
          export PATH=$PATH:$HOME/bin:$HOME/go/bin
          make github
```

Reply:
Yes, and updating the actions too. (commit inbound)